VirtoCommerce.PricingModule.Data/Services/PricingServiceImpl.cs

```diff
@@ -142,20 +149,35 @@ public virtual IEnumerable<coreModel.Price> EvaluateProductPrices(coreModel.Pric
             {
                 //Get a price range satisfying by passing context
                 var query = repository.Prices.Include(x => x.Pricelist)
-                                             .Where(x => evalContext.ProductIds.Contains(x.ProductId))
-                                             .Where(x => evalContext.Quantity >= x.MinQuantity || evalContext.Quantity == 0);
+                    .Where(x => evalContext.ProductIds.Contains(x.ProductId))
+                    .Where(x => evalContext.Quantity >= x.MinQuantity || evalContext.Quantity == 0);
+
+                if (AllowTimeFilters)
+                {
+                    evalContext.CertainDate = evalContext.CertainDate ?? DateTime.UtcNow;
```

Comment:
Use default value for certain date - it already breaking changes because `EvaluatePriceLists` method used this property for price list filtration

Reply:
I made sure context isn't changed anymore.